### PR TITLE
Add configure/kubectl-apply-all.sh

### PR DESCRIPTION
**Pull request by ggilmore**
This adds a configure/kubectl-apply-all script that basically serves as a list of all the deployments that you have added to the cluster (base+lang servers + SSD + etc) and is an easy way to run the `kubectl apply` commands to configure them

### CODE REVIEW

File: configure/jaeger/README.md [hunk 1]
```diff
@@ -2,13 +2,20 @@
 
 [Jaeger](https://github.com/jaegertracing/jaeger) is an open-source request tracing system that can run inside of your Kubernetes cluster.
 
-1.  Apply the Jaeger services to your cluster.
+1.  Append the `kubectl apply` command for the Jager resources to `configure/kubectl-apply-all.sh`.
```
Comment (keegancsmith) [inline configure/jaeger/README.md]: s/Jager/Jaeger/g
Also happens elsewhere in the doc.

File: configure/jaeger/README.md [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
 
     ```bash
+    # configure/kubectl-apply-all.sh
     kubectl apply --prune -l deploy=jaeger -f configure/jaeger --recursive
     ```
 
-2.  Add the Jager collector agent to supported services.
+1.  Apply your changes to Prometheus to the cluster.
```
Comment (keegancsmith) [inline configure/jaeger/README.md]: s/Prometheus/Jaeger/

File: configure/kubectl-apply-all.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# This file should contain `kubectl apply` commands for all of your configured resources.
+# 
+# This file should be run:
+#   * When the cluster is first created
+#   * Whenever the configuration for any resource has been updated
+
+kubectl apply --prune -l deploy=sourcegraph -f base --recursive
```
Comment (keegancsmith) [inline configure/kubectl-apply-all.sh]: we also need to make sure this command is run from the top-level repo dir. I think this script should probably live top-level directory given it is doing something to base (and in the future to configure, etc)

File: configure/jaeger/README.md [hunk 4]
```diff
@@ -2,13 +2,20 @@
 
 [Jaeger](https://github.com/jaegertracing/jaeger) is an open-source request tracing system that can run inside of your Kubernetes cluster.
 
-1.  Apply the Jaeger services to your cluster.
+1.  Append the `kubectl apply` command for the Jaeger resources to `configure/kubectl-apply-all.sh`.
 
     ```bash
+    # configure/kubectl-apply-all.sh
```
Comment (keegancsmith) [inline configure/jaeger/README.md]: same feedback for the rest and we talked in person, but rather make it a command to make this clearer

```
echo kubectl apply --prune -l deploy=jaeger -f configure/jaeger --recursive >> configure/kubectl-apply-all.sh
```

**Comment by ggilmore** [comment]
@nicksnyder @keegancsmith updated, PTAL

**Pull request merged**